spec/lib/velum/secrets_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "spec_helper"
+require "velum/secrets"
+
+describe "create_secret_key_base" do
+  key_base_dir = File.join("/tmp", "velum_secrets_spec")
```

Comment:
If you're already using `Pathname` below you could also use it here, and replace the `IO.read` ocurrences below